### ISSUE TITLE
Snooze not working on alarms set to "On Specified Week Days."

### ISSUE THREAD

**Issue by half-elfbard** [vicolo-dev/chrono#380]
**Describe the bug**
When an alarm is set to run "On Specified Week Days," the snooze function does not work as intended. Snoozing the alarm does not create the snoozed notification, and the alarm will not go off again. Inside the app, the alarm will show a stats as "Snoozed until XX:XX" as expected, but this time will pass with nothing happening. 

**To Reproduce**

1. Create a new alarm.
2. Set it to run "On Specified Week Days."
3. Set a snooze on the alarm.
4. When alarm goes off try and snooze the alarm.

**Expected behavior**
Alarm snoozes correctly when set "On Specified Week Days."

**Smartphone Information**
 - Device: Samsung S24+
 - OS: One UI 6.1 / Android 14
 - App version: app reports 0.6.0-beta1, though beta2 apk was used to install

**Comment by vmichals** [comment]
I have the same issue 

Smartphone Information

Device: Moto G Stylus (1st generation)
OS: Android 11
App version: app reports 0.6.0-beta1, installed via fdroid

**Comment by half-elfbard** [comment]
Issue persists after 0.6.0 update.

**Comment by AhsanSarwar45** [comment]
Yes, the fix wasn't pushed in 0.6.0. But I have fixed the issue and it should be out in 0.6.1.

**Comment by mtalexan** [comment]
Any chance we can get the release with this fix in it released? The fix has been waiting 3.5 months for the next release, and I'd love to be able to use Snooze on my weekday alarms.
